### PR TITLE
change task rebalancing threshold as well

### PR DESCRIPTION
**Pull request by dim-geo**
Zen also changes default task rebalancing threshold
https://github.com/zen-kernel/zen-kernel/commit/05429691ac69cfe6704530bfed1d5911efb828bb

### CODE REVIEW

File: set-cfs-zen-tweaks.sh [hunk 1]
```diff
@@ -43,6 +43,7 @@ MIN_GRANULARITY_NS_FILE="/sys/kernel/debug/sched/min_granularity_ns"
 WAKEUP_GRANULARITY_NS_FILE="/sys/kernel/debug/sched/wakeup_granularity_ns"
 MIGRATION_COST_NS_FILE="/sys/kernel/debug/sched/migration_cost_ns"
 BANDWIDTH_SIZE_US_FILE="/proc/sys/kernel/sched_cfs_bandwidth_slice_us"
+NR_MIGRATE_FILE="/sys/kernel/debug/sched/nr_migrate"
 
 if [ ! -f "$LATENCY_NS_FILE" ]; then
     echo "Detected kernel <5.13. Using legacy locations."
```
Comment (igo95862) [inline set-cfs-zen-tweaks.sh:49]: What about legacy `/proc` location?

Comment (igo95862) [reply set-cfs-zen-tweaks.sh:49]: It might be `/proc/sys/kernel/sched_nr_migrate`.

**Review by igo95862 — CHANGES_REQUESTED** [review]

File: set-cfs-zen-tweaks.sh (hunk 1 above)
Comment (dim-geo) [reply set-cfs-zen-tweaks.sh:49]: I don't have a lower kernel available to confirm

Comment (igo95862) [reply set-cfs-zen-tweaks.sh:49]: I will try a VM with older kernel this weekend and merge.

**Comment by simonvanderveldt** [comment]
@dim-geo The linked zen-kernel commit also removed the migration cost override. Shouldn't that also be removed from `cfs-zen-tweaks.sh`?

**Pull request merged**